cmd/kogito/command/flag/build_flag.go

```diff
@@ -28,6 +28,8 @@ const (
 	LocalFileResource ResourceType = "LocalFileResource"
 	// LocalDirectoryResource Build using directory on local system
 	LocalDirectoryResource ResourceType = "LocalDirectoryResource"
+	// LocalBinaryDirectoryResource Build using directory on local system
+	LocalBinaryDirectoryResource ResourceType = "LocalBinaryDirectoryResource"
```

Comment:
What is the difference between `LocalBinaryDirectoryResource` and `LocalDirectoryResource` above? They have same comment.

Reply:
Does `LocalBinaryDirectoryResource` reference the folder contained built Kogito application

Reply:
Good point, @sutaakar